packages/config/src/reader.test.ts

```diff
@@ -35,6 +35,7 @@ const DATA = {
     strings: ['string1', 'string2'],
   },
   nestlings: [{ boolean: true }, { string: 'string' }, { number: 42 }] as {}[],
+  port: 'number',
```

Comment:
This bit seems a bit strange?

Reply:
Yeah, I'm pre-coffee. I thought this was schema. I'll change it to something else.

Reply:
Coding in the Danger Zone 🎶

Reply:
![](https://media.giphy.com/media/xGbA1gRCAj1jW/source.gif)

I removed that line. It's not really doing anything.